### PR TITLE
feat(console): add console UI for private keys rotation

### PR DESCRIPTION
**Pull request by charIeszhao**
## Summary

Implement console UI to rotate OIDC private keys and cookie keys. (2/2 parts)

## Testing

Locally tested

Screenshot

## Checklist

~~- [ ] `.changeset`~~ Add later
~~- [ ] unit tests~~
~~- [ ] integration tests~~ later
~~- [ ] docs~~ later

OR

- [x] This PR is not applicable for the checklist

### CODE REVIEW

**Review by wangsijie — APPROVED** [review]

File: packages/console/src/ds-components/Table/index.module.scss
Comment (darcyYe) [inline packages/console/src/ds-components/Table/index.module.scss:1]: not sure these changes will affect other use cases of this component, should double-check?

File: packages/console/src/pages/TenantSettings/TenantBasicSettings/index.tsx [hunk 1]
```diff
@@ -114,6 +115,7 @@ function TenantBasicSettings() {
         <FormProvider {...methods}>
           <div className={styles.fields}>
             <ProfileForm currentTenantId={currentTenantId} />
+            <SigningKeys />
```
Comment (darcyYe) [inline packages/console/src/pages/TenantSettings/TenantBasicSettings/index.tsx:118]: no feature switch needed?

File: packages/console/src/pages/TenantSettings/TenantBasicSettings/SigningKeys/index.tsx [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+import useApi, { type RequestError } from '@/hooks/use-api';
+
+import * as styles from './index.module.scss';
+
+function SigningKeys() {
+  const api = useApi();
+  const { t } = useTranslation(undefined, { keyPrefix: 'admin_console.tenants.signing_keys' });
+  const [activeTab, setActiveTab] = useState<LogtoOidcConfigKey>(LogtoOidcConfigKey.PrivateKeys);
+  const keyType = activeTab === LogtoOidcConfigKey.PrivateKeys ? 'private-keys' : 'cookie-keys';
+  const entities = activeTab === LogtoOidcConfigKey.PrivateKeys ? 'tokens' : 'cookies';
```
[the remark this answers is not in the document]
Comment (charIeszhao) [reply packages/console/src/pages/TenantSettings/TenantBasicSettings/SigningKeys/index.tsx:31]: No, I think `tokens` and `cookies` do not need translation.

File: packages/console/src/ds-components/Dropdown/index.tsx [hunk 3]
```diff
@@ -49,14 +50,15 @@ function Dropdown({
...
   hasOverflowContent,
 }: Props) {
   const overlayRef = useRef<HTMLDivElement>(null);
 
   const { position, positionState, mutate } = usePosition({
-    verticalAlign: 'bottom',
+    verticalAlign,
     horizontalAlign,
-    offset: { vertical: 4, horizontal: 0 },
+    offset: { vertical: verticalAlign === 'bottom' ? 4 : -4, horizontal: 0 },
```
[the remark this answers is not in the document]
Comment (charIeszhao) [reply packages/console/src/ds-components/Dropdown/index.tsx:61]: Yeah, if the dropdown content is popped up under the select field, the vertical offset is 4, but if the dropdown is popped up above the field, then offset is -4.

File: packages/console/src/ds-components/Dropdown/index.module.scss [hunk 4]
```diff
@@ -28,4 +28,5 @@
   background: transparent;
   position: fixed;
   inset: 0;
+  z-index: 102;
```
[the remark this answers is not in the document]
Comment (charIeszhao) [reply packages/console/src/ds-components/Dropdown/index.module.scss:31]: Good point. But sadly the generic used modal dialog has a `z-index: 101`. And now we have a select box in the modal dialog, so the dropdown content of the select box should have z-index value greater than `101`

File: packages/console/src/ds-components/Table/index.module.scss
Comment (charIeszhao) [reply packages/console/src/ds-components/Table/index.module.scss:1]: Have double checked, and this is by the latest design system. So the current implementation are all inconsistent with the design. Thus, changing the ds-component styles to match the latest design.

**Pull request merged**